CMakeLists.txt

```diff
@@ -28,6 +28,7 @@ if (MSVC)
         #/wd4503      # decorated name length exceeded, name was truncated
         /W4           # enable all warnings
         /MP           # Multi-processor compilation
+        /source-charset:ISO-8859-1 # base64.hpp is in ISO-8859-1 encoding
```

Comment:
We should just fix the source file to not have high-ASCII chars

Reply:
Probably true. In this case, the non-ASCII character is (twice) in the copyright notice for the source pulled from http://www.adp-gmbh.ch/cpp/common/base64.html, and is that way in the original too, down to the encoding. It only complains if your local code page is non-European (cp932 in my case).